.github/integration/tests/sda/10_upload_test.sh

```diff
@@ -61,38 +61,4 @@ if [ "$num_log_rows" -ne 12 ]; then
     exit 1
 fi
 
-## test with token from OIDC service
-echo "testing with OIDC token"
-newToken=$(curl http://oidc:8080/tokens | jq '.[0]')
```

Comment:
I haven't tested, but why wouldn't 
```
token="$(cat /shared/token)"
```
work here instead, same way as was done for the other bash scripts in the PR?

Reply:
Why would you test with the same token twice?

Reply:
I should have explained better. Why not create a new token same way as the file `token` is created under `/shared` and use this here instead.

Reply:
Ok, I have no strong opinion about whether we keep this test or not. I agree, it does seem superfluous.